README.md

```diff
@@ -5,3 +5,128 @@
 [Exquisite corpse](https://en.wikipedia.org/wiki/Exquisite_corpse), also known as exquisite cadaver (from the original French term cadavre exquis), is a method by which a collection of words or images is collectively assembled. Each collaborator adds to a composition in sequence, either by following a rule (e.g. "The adjective noun adverb verb the adjective noun." as in "The green duck sweetly sang the dreadful dirge.") or by being allowed to see only the end of what the previous person contributed.
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Learning Goals
+
+- Practice using events
+- Pass custom information to an event handler function
+- Dynamically construct components based on data
+- Extend code provided
+
+## Objective
+
+Our goal is to create an implementation of an Exquisite corpse game for poetry in React. We will make a single-page app that will allow users to contribute to a poem one line at a time using a form. Each line of poetry is a piece of data that should propagate correctly from component to component. A user who is filling out the form may only see the most recent line of poetry submitted. When the users are finished submitting lines of poetry, they should be able to see the full poem.
```

Comment:
Maybe add a link which describe the Exquisite corpse game.  I wonder if this might have some of the same issues we'd had with Scrabble.